Content.Shared/Random/RandomHelperSystem.cs

```diff
@@ -20,7 +20,7 @@ public void RandomOffset(EntityUid entity, float minX, float maxX, float minY, f
         var offset = new Vector2(randomX, randomY);
 
         var xform = Transform(entity);
-        _transform.SetLocalPosition(xform, xform.LocalPosition + offset);
+        _transform.SetLocalPosition(entity, xform.LocalPosition + offset, xform);
```

Comment:
```suggestion
        _transform.SetLocalPosition(entity, xform.LocalPosition + offset);
```
let the method handle getting the component and delete the line above.

Reply:
This one needs the xform for the `LocalPosition` to get the right position to set

Reply:
Oh, right!